containers.go

```diff
@@ -293,6 +300,9 @@ func (r *containerStore) Create(id string, names []string, image, layer, metadat
 	if _, idInUse := r.byid[id]; idInUse {
 		return nil, ErrDuplicateID
 	}
+	if options.MountOpts != nil {
+		options.Flags["MountOpts"] = options.MountOpts
```

Comment:
Please use `append([]string{}, options.MountOpts...)` here, so that the value we set in `Flags` is a copy of the passed-in `options.MountOpts` slice, and any changes made to the slice in `options.MountOpts` after we're called won't be unexpectedly made in our map.

Reply:
Fixed